librz/core/core.c

```diff
@@ -1685,7 +1685,6 @@ RZ_API void rz_core_autocomplete(RZ_NULLABLE RzCore *core, RzLineCompletion *com
 			if (!len || !strncmp (buf->data + chr, sdbkv_key (kv), minlen)) {
 				char *p = strchr (buf->data + chr, '.');
 				if (p) {
-					j += autocomplete_pfele (core, completion, sdbkv_key (kv), pfx, j, p + 1);
```

Comment:
is this really dead? maybe the assignment is useless, but the call to `autocomplete_pfele` seems relevant to me.

Reply:
i will check at lunch